xml/templates/drop.tpl

```diff
@@ -8,11 +8,12 @@
 --
 -- Generated from {$smarty.template}
 -- {$generated}
---{/if}
+--
+{/if}
 -- /*******************************************************
 -- *
--- * Clean up the existing tables{if !$isOutputLicense} - this section generated from {$smarty.template}
-{/if}
+-- * Clean up the existing tables{if !$isOutputLicense} - this section generated from {$smarty.template}{/if}
+
```

Comment:
I don't know why the extra line is needed. I've tried several versions and this works for both cases (with or without license)...
Maybe someone can explain.